PxAPI-2.yml

```diff
@@ -124,6 +158,13 @@ components:
       required: true
       schema:
         type: string
+    codeListId:
+      name: codeListId
+      in: path
+      description: Identiyer for a code 

```

Comment:
description: Identiyer for a code  
should be 
description: Identiyer for a codeList
?

Reply:
there is a missing f there :-)

Reply:
Good catch I have updated it.